### DISCUSSION TITLE
NZBGet v24 Client Release

### DISCUSSION THREAD

**Discussion by luckedea** [nzbgetcom/nzbget#269] [Announcements]
The v24 release of NZBGet introduces a suite of updates and new features focused on improving user experience and expanding system support. Notable enhancements include a new dark theme, extended platform compatibility, and improved configuration management.  

[Download NZBGet v24 Client Here](https://nzbget.com/download/)  

### User Interface Enhancements:

* Implemented a dark theme with new icons to improve visual comfort and interface aesthetics ([\#214](https://github.com/nzbgetcom/nzbget/commit/16ecaa5c3eb2efc43e965b796a4c75cdf1d959da)).
* Added support for macOS x64 builds, compatible with macOS Mojave 10.14 and later ([\#194](https://github.com/nzbgetcom/nzbget/commit/90a89f8fb3b81432e192203e8a66419cfd3cd723)).
* Refined the NewsServer Add UI, setting encryption as the default and adjusting ports based on security settings ([\#225](https://github.com/nzbgetcom/nzbget/commit/cd1cab44b6052c3b02f14689fbafb65505f67a4e)).

### System and Packaging:

* Introduced DEB and RPM package support to simplify installation across various Linux distributions ([\#230](https://github.com/nzbgetcom/nzbget/commit/1de712a428c031513431d0d75db7ca5ac51d3caa)).
* Added support for installation via Homebrew, enhancing accessibility for macOS users.
* NZBGet v24 is now available on Windows through Winget, facilitating smoother installation and updates.
* Integrated support for QNAP devices via the Sherpa package manager, extending our reach on network-attached storage devices.

### Extension Manager and Extensions:

* Enhanced the Extension Manager with better error messages and integrated a 7-Zip exit codes decoder ([\#166](https://github.com/nzbgetcom/nzbget/commit/d11ed84912fc486e6f717e56ccd973526920c0db)).
* Updated extensions including PasswordDetector, EasySort, ExtendedUnrar, NotifyPlex, and introduced NotifySickChill as a new extension.
* Addressed various issues in existing extensions like FakeDetector and VideoSort, improving stability and compatibility.
* Fulfilled our commitment from the previous release by adding the Completion extension, enhancing download completion management.

### Developer Tools and Documentation:

* Transitioned to CMake for building processes, deprecating Autotools and MSBuild to enhance cross-platform development efficiency ([\#182](https://github.com/nzbgetcom/nzbget/commit/56e4225fc73a6d1c7cdc6f647a4c75cdf1d959da)).
* Updated and reorganized documentation to aid developers and users, including detailed installation guides and extensions usage ([\#199](https://github.com/nzbgetcom/nzbget/commit/c93b551b1fa0afae458c1e78485c6d3eb6410514)).

### Bug Fixes:

* A lot of various bugs fixed and smaller improvements added

## Community Contributions and Acknowledgements

Thank you to all our contributors for making this release the most collaborative yet, with a record number of community contributions. Your efforts are crucial in helping us refine NZBGet and keeping its legacy alive. Whether it’s providing feedback, suggesting features, or testing new updates, every contribution helps NZBGet improve. 

We are committed to actively addressing all issues and feature requests shared on [our GitHub](https://github.com/nzbgetcom). Let’s continue to work together to make NZBGet the best it can be.

**Comment by mkanet** [comment]
Thank you so much for maintaining NZBGet; and also supporting Windows OS.  Now I dont have to worry about switching to Sabnzbd.
